.github/workflows/kv-test-failure-stale.yml

```diff
@@ -0,0 +1,34 @@
+name: Mark stale test failure issues for KV
+
+on:
+  schedule:
+  - cron: "0 10 * * 1-4"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/stale@v3
+      with:
+        operations-per-run: 1000
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: |
+           We have marked this test failure issue as stale because it has been 
+           inactive for 1 month. If this failure is still relevant, removing the 
+           stale label or adding a comment will keep it active. Otherwise, 
+           we'll close it in 5 days to keep the terst failure queue tidy. 
+        stale-pr-message: 'Stale pull request message'
+        stale-issue-label: 'no-test-failure-activity'
+        stale-pr-label: 'no-pr-activity'
+        close-issue-label: 'X-stale'
+        close-pr-label: 'X-stale'
+        # Disable this for PR's, by setting a very high bar
+        days-before-pr-stale: 99999
+        days-before-issue-stale: 30
```

Comment:
Is it worth doing stale then close? We can just close right away, if someone's going to react to the notification they can reopen it. Doubling the number of notifications doesn't seem that useful.

Reply:
The bot does not support that, it does a 2 pass I am not seeing a way to turn this off.

Reply:
Ok let's live with it then.

Reply:
According to https://github.com/actions/stale/pull/64, I  think if you set `days-before-close:0`, the issue should be closed right when it is marked as stale. Let's risk it?